hwdetect/hwdetect.sh

```diff
@@ -47,6 +47,8 @@ else
 fi
 echo
 
+trap 'rm -f $usb0 $usb1 $flash0 $flash1 $i2c0 $i2c1 $sd0 $sd1 && echo -e "\n***********Detection ended***********" && exit' 2 3 15
```

Comment:
This line is too long. Is it possible to fit it to 80 columns by adding some variable like ALL_TMP_FILES or/and MSG_TEXT?